rust/src/shards/gui/context.rs

```diff
@@ -174,11 +175,18 @@ impl Shard for EguiContext {
     self.contents.warmup(ctx)?;
     self.main_window_globals.warmup(ctx);
     self.parents.warmup(ctx);
+
     // Initialize the parents stack in the root UI.
     // Every other UI elements will reference it and push or pop UIs to it.
-    self.parents.set(Seq::new().as_ref().into());
+    if !self.parents.get().is_seq() {
```

Comment:
I'm not sure if this was an issue but I expect this might clear the stack if some nested UI were to be warmed up from within a (Setup) block in another UI